core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -213,11 +215,12 @@ var ExplorationEditorTranslationTab = function() {
     await waitFor.elementToBeClickable(
```

Comment:
Here and elsewhere in your PR, you can remove the "elementToBeClickable" statement if the following "action.click" refers to the same element. This is because action.click internally does a waitFor element to be clickable.

Reply:
ok thank you

Reply:
done